src/integrations/Appcues/browser.js

```diff
@@ -21,7 +23,14 @@ class Appcues {
 
   init() {
     logger.debug('===in init Appcues===');
-    ScriptLoader('appcues-id', `https://fast.appcues.com/${this.accountId}.js`);
+
+    let url = `https://fast.appcues.com/${this.accountId}.js`;
+    if (isDefinedAndNotNullAndNotEmpty(this.proxyUrl) && typeof this.proxyUrl === 'string') {
+      url = this.proxyUrl.includes(`${this.accountId}.js`)
```

Comment:
Instead of checking for the `accountId.js`, it's better we check if the URL is either a URL prefix (`https://cdn.mycustomdomain/my/custom/path/`) or the full URL (`https://cdn.mycustomdomain/my/custom/path/accountId.js`).

This will work even if they self-host the file under a different name.

Reply:
If it's a full URL, you use it directly without appending the `${accountId}.js` again.

Reply:
updated